packages/adblocker/src/lists.ts

```diff
@@ -24,9 +28,15 @@ export const enum FilterType {
  * performed before calling a more specific parser to create an instance of
  * `NetworkFilter` or `CosmeticFilter`.
  */
-export function detectFilterType(line: string): FilterType {
+export function detectFilterType(
+  line: string,
+  { extendedNonSupportedTypes }: { extendedNonSupportedTypes?: boolean } = {},
```

Comment:
Could this be made "extendedNonSupportedTypes = false", so it has type boolean, not "boolean|undefined"?

```suggestion
  { extendedNonSupportedTypes = false }: { extendedNonSupportedTypes: boolean } = {},
```